test/unit/core/adapters/configuration.ts

```diff
@@ -150,6 +150,97 @@ suite('Configuration Adapter', ({ expect, stub }) => {
     });
   });
 
+  describe('extractPageSizes()', () => {
+    it('should do nothing if state is not an object', () => {
+      const pageSize = false;
+      const defaultValue = 0;
+
+      expect(Adapter.extractPageSizes(<any>{
```

Comment:
expectations shouldn't be this long, if it can't be fit in a couple of lines then break the action and the expectation up so you end up with
```js
const pageSizes = Adapter.extractPageSizes(<any>{ search: { pageSize } }, defaultValue);

expect(pageSizes).to.eql({ selected: 0, items: [defaultValue] });
```